src/components/structures/auth/SoftLogout.tsx

```diff
@@ -115,30 +117,32 @@ export default class SoftLogout extends React.Component<IProps, IState> {
         const queryParams = this.props.realQueryParams;
         const hasAllParams = queryParams && queryParams['loginToken'];
         if (hasAllParams) {
-            this.setState({ loginView: LOGIN_VIEW.LOADING });
+            this.setState({ loginView: LoginView.Loading });
             this.trySsoLogin();
             return;
         }
 
         // Note: we don't use the existing Login class because it is heavily flow-based. We don't
         // care about login flows here, unless it is the single flow we support.
         const client = MatrixClientPeg.get();
-        const flows = (await client.loginFlows()).flows;
-        const loginViews = flows.map(f => FLOWS_TO_VIEWS[f.type]);
+        const flows = (await client.loginFlows()).flows as [any];
```

Comment:
Could we maybe add somet typing here? It looks like `client.loginFlows` types are non-existent, and it might be a good opportunity to add them?

Reply:
err, this slipped through my diff review net. `loginFlows()` isn't properly typed, and I forgot what the function to search within an array was so manually typed it 😅 

will remove it from here